open-machine-learning-jupyter-book/ml-advanced/clustering/introduction-to-clustering.ipynb

```diff
@@ -1030,11 +1030,12 @@
    "source": [
```

Comment:
<p>Your turn is empty, maybe you forgot to associate the assignment?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>